invader_payment_paypal/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+REST Services for Paypal Payments (base module without controller).
```

Comment:
@sebastienbeau We should describe the way to use this API and how the JS is expected to interact with Odoo. We should to the same for each kind of payment. (with jscode snippets ....)